### PR TITLE
Improve display method

### PR DESCRIPTION
**Pull request by svenvanderburg**
`display` now shows this:
image

`display_rdf` shows this:
image
That also works for steps.

I updated the `noodles_fw` notebook accordingly.

### CODE REVIEW

File: fairworkflows/rdf_wrapper.py [hunk 1]
```diff
@@ -309,6 +312,30 @@ def _publish_as_nanopub(self, use_test_server=False, **kwargs):
...
+                              'to install graphviz python package. '
+                              'Version 0.14.1 is known to work well.')
+
+    def display_rdf(self):
+        graphviz = self._import_graphviz()
+
+        with TemporaryDirectory() as td:
+            filename = Path(td) / 'dag.dot'
+            with open(filename, 'w') as f:
+                rdf2dot(self._rdf, f)
```
Comment (raar1) [inline fairworkflows/rdf_wrapper.py:336]: Given that `rdf2dot` uses the now deprecated `cgi` stuff that was renamed in newer python versions, could we introduce a version check here to raise an exception (or warning) that the `display_rdf()` method won't work (I think it fails if you try to use `3.8.0` and above. The issue has been fixed, see e.g.

<https://github.com/RDFLib/rdflib/issues/1110>

But the latest release of `rdflib` doesn't have it yet, and they don't seem to release super often...

File: tests/test_fairworkflow.py [hunk 2]
```diff
@@ -251,17 +251,17 @@ def test_draw_with_graphviz_module_and_dependency(self, tmp_path, test_workflow)
...
 
-    def test_display_with_graphviz_module_and_dependency(self, test_workflow):
+    def test_display_rdf_with_graphviz_module_and_dependency(self, test_workflow):
         """
-        Test display method with graphviz python module and graphviz software
+        Test display_rdf method with graphviz python module and graphviz software
         installed
         """
-        test_workflow.display(full_rdf=True)
+        test_workflow.display_rdf()
```
Comment (raar1) [inline tests/test_fairworkflow.py:264]: Likewise, maybe we could skip these tests if the python version is `3.8.0` and above?

**Review by raar1 — APPROVED** [review]
Good work - these both work/look like how I wanted them to. I've left two suggestions below

File: tests/test_fairworkflow.py (hunk 2 above)
Comment (svenvanderburg) [reply tests/test_fairworkflow.py:264]: I created a new issue for this: #182

File: fairworkflows/rdf_wrapper.py (hunk 1 above)
Comment (svenvanderburg) [reply fairworkflows/rdf_wrapper.py:336]: I created a new issue for this: #182

**Pull request merged**